### ISSUE TITLE
There's no way to add input fileds (arguments) to queries

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [apollographql/apollo-studio-preview-docs#1]
Before (current):

![before](https://user-images.githubusercontent.com/7892779/82143225-f5f99d80-984a-11ea-8495-1970608623e0.png)

After (proposed):

![after](https://user-images.githubusercontent.com/7892779/82143235-10cc1200-984b-11ea-9c16-b75b3da4eb9c.png)

**Comment by daniman** [comment]
@FluorescentHallucinogen this is a great idea! We're still working on handling arguments and input objects fully in the search and docs, but it's in active development.

I'll pass your thought here on to @Jephuff who has been working on handling arguments in particular.

**Comment by FluorescentHallucinogen** [comment]
> We're still working on handling arguments and input objects

What about GraphQL list (array) inputs? This is essential feature. See https://github.com/OneGraph/graphiql-explorer/issues/10#issuecomment-567901798. :wink:

**Comment by FluorescentHallucinogen** [comment]
Re handling arguments in the search. It would be really nice to have advanced search options to include/exclude arguments to/from search results. Something like this:

![graphql-advanced-search](https://user-images.githubusercontent.com/7892779/82187433-12eaab00-98f5-11ea-8548-393b917e602b.png)

**Comment by FluorescentHallucinogen** [comment]
Re "Search in descriptions". Sometimes user doesn't remember exactly the name of the field/argument, but remembers some keyword from its description.

**Comment by jglovier** [comment]
@FluorescentHallucinogen thanks for your thoughtful suggestion (both the argument adding and the search options).

Per the advanced search options, currently we are thinking of expanding the search syntax to afford for use cases like arguments. But eventually we may add form based search options like your suggestion. We'll certainly keep this in mind as we progress! ✨ 🙇✨

**Comment by FluorescentHallucinogen** [comment]
> We're still working on handling arguments and input objects fully in the search and docs

> Per the advanced search options, currently we are thinking of expanding the search syntax to afford for use cases like arguments.

As an idea, what about search filter chips? :wink:

![graphql-search-filter-chips](https://user-images.githubusercontent.com/7892779/82762653-e65df400-9e0a-11ea-87ba-eb67c0630299.png)

**Comment by FluorescentHallucinogen** [comment]
See https://graphqleditor.com/static/explorer-ba1e02884f0676ce1eb1e67c041e63dc.mp4

**Comment by jglovier** [comment]
Yeah, that's a really cool experience for filtering available fields!

**Comment by daniman** [comment]
@FluorescentHallucinogen I've renamed this issue because we're on the cusp of landing exactly what you described with the arguments, so I figured we could refocus the rest of this issue on Search.
